src/main/java/application/fxobjects/cell/graph/BubbleCell.java

```diff
@@ -75,6 +75,21 @@ public void focus() {
     }
 
     /**
+     * Method to set the side focus.
+     */
+    public void sideFocus() {
```

Comment:
Can't this code be implemented in an interface where each Cell inherits the sideFocus behaviour? As copy-pasting code in multiple classes is not very good for maintainibility.

See also: http://howtodoinjava.com/object-oriented/multiple-inheritance-in-java/